scripts/report_integration_test.js

```diff
@@ -0,0 +1,101 @@
+const fs = require('fs');
```

Comment:
This code is not maintainable. It's basically parsing the output of running our tests and trying to find information from that. If `jest` changes just a tiny bit their output this will break.

Maybe we can look at how the code coverage pull metrics and use something like that to find the tests that worked.

Reply:
Somehow I missed the "test result" part of the payload when I first tried.
I'll switch.